package.json

```diff
@@ -91,47 +77,50 @@
   "dependencies": {
     "global": "^4.3.2",
     "three": "0.92.0",
-    "video.js": "^6.2.7",
+    "video.js": "6.10.2 || ^7.0.3",
```

Comment:
why 6.10.2 specifically? probably should stay `^6.2.7`.

Reply:
Don't ask me Mate, im absolutely don't understand what should i do with thats scripts and etc 

Just im going to find any information or files, what my wife masked from me .
Because im very tired founding parts of files from cloud platform and certificate .

I know what she has sent money to Polish programmer but this is not enough .

I don't understand which secrets she has if ready pay for that money .